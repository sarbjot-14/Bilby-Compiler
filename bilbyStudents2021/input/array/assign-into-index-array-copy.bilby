main{
	imm integers := [11,12,13,14,15];
	
	imm integers_copy := integers;
	
    print integers_copy $n;
    
    integers_copy[2] := 22222222;
    
    
    print integers $n;
    
    print integers_copy $n;
   

}