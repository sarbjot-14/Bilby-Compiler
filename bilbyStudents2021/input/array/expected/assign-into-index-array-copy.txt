[11, 12, 13, 14, 15]
[11, 12, 22222222, 14, 15]
[11, 12, 22222222, 14, 15]
